.github/workflows/ci-contracts.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
```

Comment:
Why is this separate from the solidity tests?

Reply:
So originally it was all one workflow but I broke it into two in order to path filter. I just fixed the name though 👍